python/python/bystro/proteomics/scripts/get_uniprot_id_gene_name_mapping.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python
+
+"""Download a list of uniprot_id / gene_name pairs from Uniprot and store the result on disk as csv."""
+
+import pandas as pd
+import tqdm
+from unipressed import UniprotkbClient  # type: ignore[import]
```

Comment:
Will this allow us to support non-human proteomes as well? Since we're supporting non-human genomes, there's a pretty good chance we might want to support non-human proteome uniprot IDs.

Reply:
The client will allow us to support non-human proteomes, yes.  We could just download everything, or at least everything for the reference genomes we support.  Given that this script is speculative development (we don't actually know that we'll need it even for human datasets), would you feel comfortable noting it as a todo for now?

Reply:
Yep, I'm happy with that being a todo for now.